packages/shared/components/popups/StakingConfirmation.svelte

```diff
@@ -29,15 +34,23 @@
     let showTooltip = false
     let tooltipAnchor: unknown
 
-    const activeAirdrops =
+    const activeAirdropsParticipatedIn =
         $selectedAccountParticipationOverview?.participations
             .map((p) => getAirdropFromEventId(p.eventId))
             // Falsy values (undefined, null, ...) are filtered out from the array
             .filter(Boolean) || []
 
+    const activeAirdrops = STAKING_EVENT_IDS.filter((x) => x).map((x) => getAirdropFromEventId(x))
+
     const airdropSelections: { [key in StakingAirdrop]: boolean } = {
-        [StakingAirdrop.Assembly]: canReachAirdropMinimum(StakingAirdrop.Assembly) ? true : false,
-        [StakingAirdrop.Shimmer]: canReachAirdropMinimum(StakingAirdrop.Shimmer) ? true : false,
+        [StakingAirdrop.Assembly]:
+            activeAirdrops.includes(StakingAirdrop.Assembly) && canReachAirdropMinimum(StakingAirdrop.Assembly)
+                ? true
```

Comment:
We can get rid of this ternary operator here, and just have `[StakingAirdrop.Assembly]: activeAirdrops.includes(StakingAirdrop.Assembly) && canReachAirdropMinimum(StakingAirdrop.Assembly)`. It's effectively the same.

Reply:
Lol why was it a ternary operator in the first place?

Reply:
I guess the same reason you left it as a ternary operator - was simply glossed over :slightly_smiling_face:.